index_settings_test.go

```diff
@@ -378,6 +380,47 @@ func TestIndex_GetSortableAttributes(t *testing.T) {
 	}
 }
 
+func TestIndex_GetTypoTolerance(t *testing.T) {
+	type args struct {
+		UID    string
+		client *Client
+	}
+	tests := []struct {
+		name     string
+		args     args
+		wantResp *TypoTolerance
+	}{
+		{
+			name: "TestIndexBasicGetTypoTolerance",
+			args: args{
+				UID:    "indexUID",
+				client: defaultClient,
+			},
+			wantResp: &defaultTypoTolerance,
+		},
+		{
+			name: "TestIndexGetTypoToleranceWithCustomClient",
+			args: args{
+				UID:    "indexUID",
+				client: customClient,
+			},
+			wantResp: &defaultTypoTolerance,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			SetUpIndexForFaceting()
+			c := tt.args.client
```

Comment:
It is a common practice to define variables like this in Go?

Reply:
Not so much, but same it's because it's autogenerated